pages/SavingsForm.js

```diff
@@ -0,0 +1,47 @@
+import AccountForm from './AccountForm';
+
+class SavingsForm extends AccountForm {
+
+    get savingsSavingsButton (){return $('label[for="Savings"]')}
+
+    get moneyMarketSavingsButton (){return $('label[for="Money Market"]')}
+  
+    get savingsNameInput (){return $('#name')}
```

Comment:
You already have this element in AccountForm, you can use that instead and you won't have to implement it here